src/core-layers/grid-cell-layer/grid-cell-layer.js

```diff
@@ -84,33 +86,14 @@ export default class GridCellLayer extends Layer {
     /* eslint-enable max-len */
   }
 
-  updateAttribute({props, oldProps, changeFlags}) {
-    if (props.fp64 !== oldProps.fp64) {
-      const attributeManager = this.getAttributeManager();
-      attributeManager.invalidateAll();
-
-      if (props.fp64 && props.coordinateSystem === COORDINATE_SYSTEM.LNGLAT) {
-        attributeManager.addInstanced({
-          instancePositions64xyLow: {
-            size: 2,
-            accessor: 'getPosition',
-            update: this.calculateInstancePositions64xyLow
-          }
-        });
-      } else {
-        attributeManager.remove(['instancePositions64xyLow']);
-      }
-    }
-  }
-
   updateState({props, oldProps, changeFlags}) {
     super.updateState({props, oldProps, changeFlags});
     // Re-generate model if geometry changed
     if (props.fp64 !== oldProps.fp64) {
       const {gl} = this.context;
       this.setState({model: this._getModel(gl)});
+      this.state.attributeManager.invalidateAll();
```

Comment:
Nit: Do they all need to be invalidated? Or is that just to force the new model to be updated?

Reply:
We use this to update all attributes of the new model. Although only the low xy attributes need to be recalculated, we do not have an easy way to trigger reset all attributes (vs. only changed attributes) in `layer.updateAttributes`.
This line is what we have been doing when switching `fp64`. Given that it's an uncommon scenario in production, the perf concern is low.